### PR TITLE
Add the troubleshooting guide

### PR DESCRIPTION
**Pull request by jgpruitt**
## Description

Adds a markdown document containing helpful information for troubleshooting issues with promscale deployments.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: docs/runbooks/troubleshooting-guide.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Troubleshooting Guide
+
+## General information to always gather
+
+This is a basic set of information that is almost always relevant to troubleshooting.
+
+* Determine the version of Postgres, TimescaleDB, the Promscale connector, and the Promscale extension.
```
Comment (alejandrodnm) [inline docs/runbooks/troubleshooting-guide.md:7]: Do you think we can provide a oneliner that returns all this information?

File: docs/runbooks/troubleshooting-guide.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+FROM timescaledb_information.chunks 
+WHERE hypertable_schema='prom_data' 
+GROUP BY is_compressed;
+```
+
+Usually, if background jobs are taking a very long time and consuming a lot of 
+resources it happened in versions 0.10.0 and earlier as the number of chunks in
+the database grew. Upgrading to 0.11.0 or later should improve things significantly. 
+
+You can check [pg_stat_statements](https://www.postgresql.org/docs/current/pgstatstatements.html)
```
Comment (alejandrodnm) [inline docs/runbooks/troubleshooting-guide.md:34]: Is this loaded by default in timescale cloud? If so, maybe we can mention it.

File: docs/runbooks/troubleshooting-guide.md (hunk 1 above)
Comment (alejandrodnm) [inline docs/runbooks/troubleshooting-guide.md:7]: Do you think we can provide a oneliner that returns all this information?

**Review by alejandrodnm — APPROVED** [review]

File: docs/runbooks/troubleshooting-guide.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Troubleshooting Guide
+
+## General information to always gather
+
+This is a basic set of information that is almost always relevant to troubleshooting.
+
+* Determine the version of Postgres, TimescaleDB, the Promscale connector, and the Promscale extension.
+  * Postgres: `select version();`
+  * Timescaledb and the Promscale extensions: `select extname, extversion from pg_extension where extname in ('timescaledb', 'promscale');`
+* Determine the chunk policy: `select _prom_catalog.get_default_chunk_interval();`
```
Comment (arajkumar) [inline docs/runbooks/troubleshooting-guide.md:10]: Can we also mention how to get chunk and it's index size to configure chunk interval appropriately?

File: docs/runbooks/troubleshooting-guide.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* Determine the retention policy: `select _prom_catalog.get_default_retention_period();`
+* Determine the infrastructure setup for TimescaleDB and Promscale. Does anything else share compute or storage resources (do other things run on the same infrastructure?)?
+* If using Prometheus (run PromQL queries against Promscale if you have multiple Prometheus instances)
+  * Number of Prometheus instances
+  * Total metric ingest rate (if you are ingesting metrics): `sum(rate(prometheus_tsdb_head_samples_appended_total[5m]))`
+  * Total active series: `prometheus_tsdb_head_series`
+  * Series churn: `sum(rate(prometheus_tsdb_head_series_created_total[5m]))`
+* If using OpenTelemetry traces:
```
Comment (arajkumar) [inline docs/runbooks/troubleshooting-guide.md:18]: OpenTelemetry/Jaeger

File: docs/runbooks/troubleshooting-guide.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  * Span ingest rate: `SELECT count(*)/600 as span_throughput FROM ps_trace.span WHERE start_time > NOW() - INTERVAL '10m';`
+
+## Background Jobs / Compression Issues
+
+Check how many chunks are compressed / uncompressed:
```
Comment (arajkumar) [inline docs/runbooks/troubleshooting-guide.md:23]: Actually this give compressed and total chunks.

**Pull request merged**